services/frontend-v3/src/components/errorResult/errorResult.jsx

```diff
@@ -36,16 +40,43 @@ const ErrorResult = ({ errorCode }) => {
         </Button>
       ),
     },
+    profileNotExist: {
+      status: "404",
+      title: <FormattedMessage id="profile.not.found" />,
+      subTitle: <FormattedMessage id="profile.not.found.description" />,
+      extra: (
```

Comment:
Do you think we should break this into different components? One for each error page and then errorResult.jsx could use the different components?  I think having each component have one task would be ideal.

Reply:
So lets say we keep the `<ErrorResultView />` and have something like `<404 />`, `<403 />`, and `<ProfileNotFound />`?

Reply:
Yeah, this would give us scalability as well since if we need to add more we could always create more components